### PR TITLE
Add myself as maintainer to new project 'typed-svg'

### PR DESCRIPTION
**Pull request by rupertlssmith**
Maintainer details added to the table.

### CODE REVIEW

File: maintainers.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ This file contains info on elm-community packages and who the current maintainer
 | [svg-extra](http://github.com/elm-community/svg-extra) | Additional functions for working with Svg | jvoigtlaender | janis.voigtlaender@gmail.com |
 | [undo-redo](http://github.com/elm-community/undo-redo) | Easy undo in Elm | rohanorton | rohan.orton@gmail.com |
 | [webgl](http://github.com/elm-community/webgl) | Functional 3D Rendering with WebGL in Elm | w0rm | unsoundscapes@gmail.com |
-
+| [typed-svg](http://github.com/elm-community/typed-svg) | TypedSvg makes it more pleasant to work with SVG. | rupertlssmith | typedsvg@thesett.com |
```
Comment (eeue56) [inline maintainers.md]: This description seems a bit off - what about "Type-safe API for SVG" instead?

**Comment by rupertlssmith** [comment]
Updated the description as suggested.

**Comment by rupertlssmith** [comment]
Just giving a gently bump. The description in the file has been changed, so should be good to merge in now? I've got some updates to add to 'typed-svg'...

**Comment by eeue56** [comment]
Nope, need to wait for @canadaduane to fork typed-svg back to their account first.

**Comment by eeue56** [comment]
- delete all the tags the repo currently has
- reset the elm-package.json file to 1.0.0
- make sure the elm-package.json points to elm-community/typed-svg
- publish as 1.0.0

**Pull request merged**